packages/stores/src/mockStore.ts

```diff
@@ -298,6 +302,15 @@ const mock = (): TStores & { is_mock: boolean } => {
             setDarkMode: jest.fn(),
             setReportsTabIndex: jest.fn(),
             has_real_account_signup_ended: false,
+            real_account_signup_target: '',
```

Comment:
could you please sort it alphabetically, once my card for language changes will be merged (I refactored sorting there)